### PR TITLE
update network endpoint and genesis block

### PR DESCRIPTION
**Pull request by v9n**
Currently, in project.yaml, we connect to production node by default at neumann.api.onfinality.io/public-ws. But this endpoint appear to be down now. Upon switching to rightg endpoint, I can see data is started to index for existing mapping.

I'm switching it to our prod RPC node default, so that out of the box if someone run this they will connect to it.
I also add an extra project-staging.yaml. I will update the app structure to eventually we can run the indexer to connect to either prod/staging rpc env.

### CODE REVIEW

**Review by chrisli30 — APPROVED** [review]
Looks great! Some context regarding the `neumann` environment 👇 

The Neumann environment is completely deprecated. It was a standalone blockchain that doesn’t require a relay chain to run, but is out-of-date due to lack of maintenance.

We have only two environments now:
Turing Staging - the testing environment before Turing. We should use this to test all new implementations. It’s connected to Rococo relay chain.
Turing Network - the production environment. It’s connected to Kusama relay chain.

File: project-staging.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+specVersion: 0.2.0
+name: neumann-subql
```
Comment (chrisli30) [inline project-staging.yaml]: should update to
turing-staging-subql

File: project-staging.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+version: 0.0.4
+description: Any queries for subquery for OAK's neumann network
```
Comment (chrisli30) [inline project-staging.yaml]: Turing Staging network

**Comment by v9n** [comment]
Thanks Chris. I corrected the name. Going to merge the PR.

**Pull request merged**